clients/java/signalr/src/main/java/ActionBase.java

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+import com.google.gson.Gson;
+import com.google.gson.JsonArray;
+
+import java.util.ArrayList;
+
+public class ActionBase {
```

Comment:
This seems really gross to me... Is there no other way we can do this? Does Java have something like reflection that would let us just apply an array of arguments to a function?

Reply:
Could we build code that wraps any of `Action1` through `Action5` in a different lambda that takes an `Object[]` and then just store that wrapper instance in the list rather than this big class-full-o-pointers?